tools/topology/sof/pipe-waves-codec-playback.m4

```diff
@@ -0,0 +1,44 @@
+# Low Latency Passthrough with Waves codec Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+# host PCM_P --> B0 --> Waves codec --> B1 --> sink DAI0
+
+ifdef(`ENDPOINT_NAME',`',`fatal_error(`Pipe requires ENDPOINT_NAME to be defined: Speakers, Headphones, etc.')')
+
+# Waves codec setup config
```

Comment:
Can you say how this is generated in the comment. i.e. to help any integrator or developer who has to tune WAVES.

Reply:
@lgirdwood 
It has a detailed description in the pipe-codec-adapter-playback.m4. The comment was updated to point there